cuqi/model/_model.py

```diff
@@ -472,19 +472,19 @@ def __repr__(self) -> str:
 
 
 class AffineModel(Model):
-    """ Model representing an affine operator, i.e. a linear operator with a fixed shift. For more details on linear operators, see :class:`~cuqi.model.LinearModel`.
+    """ Model class representing an affine model, i.e. a linear operator with a fixed shift. For linear models, represented by a linear operator only, see :class:`~cuqi.model.LinearModel`.
 
     Parameters
     ----------
 
-    linear_operator : 2d ndarray or callable function.
+    linear_operator : 2d ndarray, callable function or cuqi.model.LinearModel
```

Comment:
Just a minor point, should we support that? (Linear model as operator). It felt a bit unnatural to me since LinearModel is subclass of AffineModel.

Reply:
Seems OK to me.